daemon/oci_linux_test.go

```diff
@@ -209,3 +216,38 @@ func TestGetSourceMount(t *testing.T) {
 	_, _, err = getSourceMount(cwd)
 	assert.NilError(t, err)
 }
+
+func TestDefaultResources(t *testing.T) {
+	skip.If(t, os.Getuid() != 0, "skipping test that requires root") // TODO: is this actually true? I'm guilty of following the cargo cult here.
+
+	c := &container.Container{
+		HostConfig: &containertypes.HostConfig{
+			IpcMode: containertypes.IPCModeNone,
+		},
+	}
+	d := setupFakeDaemon(t, c)
+
+	s, err := d.createSpec(context.Background(), &configStore{}, c)
```

Comment:
```suggestion
	s, err := d.createSpec(context.Background(), c)
```
- Because the 24.0 branch doesn't have #43980

Reply:
oh! thanks for the reminder; forgot about this one; just pushed an update with your suggestion🤞